### ISSUE TITLE
More informations

### ISSUE THREAD

**Issue by benda95280** [jasonwynn10/DimensionAPI#1]
Dear Jasonwynn10,

Is it possible to have more information about it ?
I'm not able to understand what we can do with it.

**Comment by Wertzui123** [comment]
Yes, please make a usefull readme, because I don't know what it does too

**Comment by Olybear9** [comment]
Lol
